webui/react/src/pages/TrialDetails/TrialDetailsLogs.tsx

```diff
@@ -171,12 +172,14 @@ const TrialDetailsLogs: React.FC<Props> = ({ experiment, trial }: Props) => {
 
   return (
     <div className={css.base}>
-      <LogViewer
-        decoder={mapV1LogsResponse}
-        title={logFilters}
-        onDownload={handleDownloadLogs}
-        onFetch={trial && handleFetch}
-      />
+      <Spinner conditionalRender spinning={!trial}>
```

Comment:
same thing here:

i think leaving out the `conditionalRender` is better for now, because it makes it more obvious what it actually is that is loading.